### PR TITLE
Add TypeScript example

### PR DESCRIPTION
**Pull request by acro5piano**
https://node-typescript-olzgjiiaef.now.sh/

### CODE REVIEW

File: node-typescript/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM mhart/alpine-node:10 as base
+WORKDIR /usr/src
+COPY . .
+RUN yarn
+RUN yarn build
```
Comment (sergiodxa) [inline node-typescript/Dockerfile]: After this step run `yarn --production` to remove devDependencies so they are not copied to the final image.

File: node-typescript/.gitignore [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+.next
+
+# vuepress build output
+.vuepress/dist
+
+# Serverless directories
+.serverless
+
+
+dist
```
Comment (sergiodxa) [inline node-typescript/.gitignore]: There is a lot of unnecessary lines here (eg. .next or .vuepress/idst), could you only let necessary ignored files for this example? node_modules is already ignored in the whole repo so just `dist` is enough I suppose.

**Comment by acro5piano** [comment]
@sergiodxa Thank you for your review! I have fixed.

**Comment by TooTallNate** [comment]
Thank you @acro5piano 🙇

**Pull request merged**